src/main/java/duke/ParsedAnswerHandler.java

```diff
@@ -0,0 +1,118 @@
+package duke;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+/**
+ * This class receives a ParsedAnswer object and execute commands based on information
+ * from the object.
+ */
+
+public class ParsedAnswerHandler {
+    ParsedAnswer pa;
+
+    ParsedAnswerHandler(ParsedAnswer pa) {
+        this.pa = pa;
+    }
+
+    void execute() {
+        String filePath = System.getProperty("user.dir")  + "/data/storage.txt";
+        switch (pa.getCommand()) {
+            case "bye":
+                System.out.println("Have a nice day.");
+                System.exit(0);
+                break;
+
+            case "list":
+                TaskList.list();
+                break;
+
+            case "todo":
+                ToDos td = new ToDos(pa.getDesc());
+                TaskList.add(td);
+                try {
+                    String sb = "T," +
+                            "1," +
+                            td.getDescription() + '\n';
+                    BufferedWriter writer = new BufferedWriter(new FileWriter(filePath, true));
+                    writer.append(sb);
+                    writer.close();
+                    System.out.println("Successfully added todo task to list.");
+                } catch (IOException e) {
+                    System.out.println("Error saving task to file.");
+                }
+                break;
+
+            case "deadline":
+                Deadline dl = new Deadline(pa.getDesc(), pa.getDate());
+                TaskList.add(dl);
+                try {
+                    String sb = "D," +
```

Comment:
According to the coding standard, line wrapping should be made after a comma or before an operator. Since `+` is an operator, you need to change to new line before it.